index.js

```diff
@@ -61,6 +62,13 @@ module.exports = {
         "mocha/no-pending-tests": "error",
         "mocha/no-return-and-callback": "error",
         "mocha/no-sibling-hooks": "error",
-        "mocha/no-async-describe": "error"
+        "mocha/no-async-describe": "error",
+
+        // verify copyright headers are present - point to templates in this eslint folder (hacky)
```

Comment:
@tmathern @jdelbick @DrGirlfriend @pheenomenon 

We need to find a way to choose the license header template between public and internal repos, if we want to use the same `@adobe/eslint-config-asset-compute` dependency everywhere.

Options:
1. create another internal eslint-config inheriting from this one here
   - pro: very clean
   - con: always need to release 2 modules for changes here
2. actually put the expected license header in each repo
   - pro: as intended by eslint (I think)
   - con: too many repos :-), a shared location would be nice

... more options?

Reply:
I'd choose 1. I don't think rules will change a lot once we're done with open-sourcing.

Reply:
Created https://jira.corp.adobe.com/browse/NUI-550